onnx/backend/test/cmd_tools.py

```diff
@@ -48,6 +48,8 @@ def prepare_dir(path: str) -> None:
                     output_dir, f'test_data_set_{i}')
                 prepare_dir(data_set_dir)
                 for j, input in enumerate(inputs):
+                    if input is None:
```

Comment:
Trying to understand this: this file is about *model* inputs/outputs, right, and not about *node/op* inputs/outputs. For model inputs/outputs there is no possibility of optional-input/output of kind 1 (static), right? I don't understand why this is being triggered. (May be some fix in the model-generator is required if it is adding an unnecessary model-input?)

Reply:
I am worried things will break if a model has an input X, and there is no corresponding input-file in the test-folder.